src/util/qhash.h

```diff
@@ -0,0 +1,5 @@
+#if QT_VERSION <= QT_VERSION_CHECK(6, 0, 0)
```

Comment:
Minor nitpick: Please use `>=` and exchange the arms. The legacy code should be in the 2nd arm. That's how we do it usually.

Reply:
Somehow that doesn't build??

```
#if QT_VERSION_CHECK(6, 0, 0) >= QT_VERSION
typedef size_t qhash_seed_t;
#else
typedef uint qhash_seed_t;
#endif
```

```
../src/track/trackref.h:137:18: error: call of overloaded ‘qHash(const TrackId&, qhash_seed_t&)’ is ambiguous
  137 |             qHash(key.getId(), seed);
      |             ~~~~~^~~~~~~~~~~~~~~~~~~
```

Reply:
`#if QT_VERSION >= QT_VERSION_CHECK(6, 0, 0)` ;)